src/tools/ScenarioMeasurement/Startup/ProcessTimeParser.cs

```diff
@@ -19,21 +19,21 @@ public void EnableUserProviders(TraceEventSession user)
         {
         }
 
-        public IList<Counter> Parse(string mergeTraceFile, string processName)
+        public IList<Counter> Parse(string mergeTraceFile, string commandLine)
```

Comment:
You'll need to change the logic in the other parsers to account for this. Perhaps we should instead record the PID from the process executions and pass in a list here?

Reply:
We could if the process run method can return a tuple of (result, pid) ?